app/views/reviews/_table_form.html.erb

```diff
@@ -0,0 +1,13 @@
+<% @review.each do |review| %>
```

Comment:
Can you please explain why you are not getting orders from review?

Reply:
This is the reviews controller and I am getting the reviews directly so, there is no need of getting orders that's why i didn't get orders from reviews.